controllers/operandrequest/reconcile_operand.go

```diff
@@ -1038,6 +1071,63 @@ func (r *Reconciler) updateK8sResource(ctx context.Context, existingK8sRes unstr
 		return nil
 	}
 
+	if kind == "Route" {
```

Comment:
I am wondering if we are able to wrap the `update` action for `Job` and `Route` into their own functions?
It would help to reduce the number of lines for the function `updateK8sResource`